### ISSUE TITLE
'User' object has no attribute 'stripe_id'

### ISSUE THREAD

**Issue by rnschmidt** [GoodCloud/django-zebra#7]
In the update view (in the marty app), where is stripe_id in user.stripe_id coming from on line 24?

https://github.com/GoodCloud/django-zebra/blob/master/zebra_sample_project/marty/views.py

I am following this example, but when I submit the payment form I get this error:

"'User' object has no attribute 'stripe_id'"

**Comment by skoczen** [comment]
Thanks, Ron. Will get this mis-example cleared up.

**Comment by caguilar-dev** [comment]
This continue without works.

Not is a good solution add a profile for this?

Best Regards

**Comment by skoczen** [comment]
Your profile model needs to subclass the StripeCustomer mixin - the view as-is doesn't work properly, I just haven't had the time to get it cleaned up.  Perhaps this weekend!

**Comment by ukch** [comment]
It's been three years and the issue is still here. Any chance of a fix?

**Comment by leetrout** [comment]
Sorry @ukch it's definitely fallen out of view. Are you actively using this library?

**Comment by ukch** [comment]
No, I'm just looking into it. Is it still maintained?

**Comment by leetrout** [comment]
No, not really. We built this when Stripe was still in beta and for various reasons no one really kept up with it. It probably wouldn't take much to bring it back in line but the architecture pattern didn't seem to appeal to the masses (take what you want vs the more common override what you don't).

I would suggest looking closer at https://github.com/pinax/pinax-stripe and https://github.com/pydanny/dj-stripe as I'm sure you already are.

On the other hand- if you want to further this library that would be a welcome effort. Unfortunately I don't have projects left that are using this in anger and I don't think @skoczen does either.
